MapboxNavigation/NavigationOptions.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import MapboxCoreNavigation
+
+/*
+    Object representing available options for a Navigation Session.
```

Comment:
Can you expand on this to avoid confusion with the similarly named NavigationRouteOptions class? Maybe we also need some disambiguation in the NavigationRouteOptions documentation too.